doc_src/history.txt

```diff
@@ -60,6 +67,16 @@ history delete --prefix "foo"
 # You can select more than one entry by entering their IDs seperated by a space.
 \endfish
 
+\subsection setting-the-histfile Setting the name of the history file
+
+By default interactive commands are logged to `$XDG_DATA_HOME/fish/fish_history` (typically `~/.local/share/fish/fish_history`).
+
+You can set the `FISH_HISTFILE` variable to a path name to change where commands are logged. It does not have to be an environment variable (despite the all uppercase name) and normally should not be an environment variable. You can change it at will within an interactive fish session to change where subsequent commands are logged. If an environment variable exists by that name when the shell starts then that file will be used rather than the default. Simply unset the variable to revert to the default history file.
```

Comment:
@faho, I agree that using "exported" is preferable to "environment" in this context. But note that the current fish docs are very inconsistent as to whether `exported` or `environment` is preferred. Do `grep 'environment var' doc_src/* | wc -l` and `grep 'exported var' doc_src/* | wc -l` and notice the former matches 48 lines while the latter matches 8 lines. 😄  The author of this PR was just copying text from my initial change to address issue #102 so he's only parroting my confusion as to the preferred nomenclature. In fact, one of our FAQ sections is titled "How do I set or clear an environment variable?".